src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -202,6 +202,11 @@ private static void fillWorkspaceWithResourcesAndData(Workspace workspace) {
 				workspace.getTagElements().get(tag).add("~CUSTOM:Exampleenchantment2");
 			}
 
+			tag = new TagElement(TagType.GAME_EVENTS, "minecraft:test");

```

Comment:
This tag does not exists, so this will crash tests when running locally with test server. I recommend testing PRs with running local tests